includes/wikia/Extensions.php

```diff
@@ -1803,7 +1803,7 @@
 // If you want to delete this extension remember to update CategoryPage3
 include("$IP/extensions/wikia/CategoryExhibition/CategoryExhibition_setup.php" );
 
-if ( !empty( $wgWatchShowURL ) ) {
+if ( !empty( $wgWatchShowURL || $wgWatchShowURLMobile ) ) {
```

Comment:
should be in a separate empty() call

Reply:
this is what javascript does to people :p